adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -240,7 +240,7 @@ class AlignmentRecordRDDFunctions(rdd: RDD[AlignmentRecord])
   def saveAsSamString(sd: SequenceDictionary,
                       rgd: RecordGroupDictionary): String = {
     // convert the records
-    val (convertRecords: RDD[SAMRecordWritable], header: SAMFileHeader) = rdd.convertToSam(sd, rgd)
+    val (convertRecords: RDD[SAMRecordWritable], header: SAMFileHeader) = convertToSam(sd, rgd)
```

Comment:
> since the RDD → AlignmentRecordRDDFunctions implicit is no longer being imported

Is that why this needs to be changed?  Since we're no longer importing the implicit ```AlignmentRecordRDDFunctions.convertToSam``` on RDDs, the method needs to be called directly.